src/WebJobs.Extensions.DurableTask/_CSharpLanguageHelpers.cs

```diff
@@ -0,0 +1,265 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
You're welcome to borrow some files that I've been using for `init`-only properties and all of the nullable attributes:
- https://github.com/wsugarman/Sweetener/blob/main/src/Shared/NullableAttributes.cs
- https://github.com/wsugarman/Sweetener/blob/main/src/Shared/IsExternalInit.cs